Exception when restoring KropView from savedInstanceState

The following exception is thrown when restoring KropView from savedInstanceState, causing the app to crash.

```
E/Parcel: Class not found when unmarshalling: com.avito.android.krop.ZoomableImageView$SavedState
    java.lang.ClassNotFoundException: com.avito.android.krop.ZoomableImageView$SavedState
        at java.lang.Class.classForName(Native Method)
        at java.lang.Class.forName(Class.java:453)
        at android.os.Parcel.readParcelableCreator(Parcel.java:2897)
        at android.os.Parcel.readParcelable(Parcel.java:2851)
        at com.avito.android.krop.KropView$SavedState.<init>(KropView.kt:263)
        at com.avito.android.krop.KropView$SavedState$CREATOR.createFromParcel(KropView.kt:268)
        at com.avito.android.krop.KropView$SavedState$CREATOR.createFromParcel(KropView.kt:266)
        at android.os.Parcel.readParcelable(Parcel.java:2860)
        at android.os.Parcel.readValue(Parcel.java:2754)
        at android.os.Parcel.readSparseArrayInternal(Parcel.java:3204)
        at android.os.Parcel.readSparseArray(Parcel.java:2416)
        at android.os.Parcel.readValue(Parcel.java:2811)
        at android.os.Parcel.readArrayMapInternal(Parcel.java:3123)
        at android.os.BaseBundle.initializeFromParcelLocked(BaseBundle.java:273)
        at android.os.BaseBundle.unparcel(BaseBundle.java:226)
        at android.os.Bundle.getSparseParcelableArray(Bundle.java:1009)
        at com.android.internal.policy.PhoneWindow.restoreHierarchyState(PhoneWindow.java:2129)
        at android.app.Activity.onRestoreInstanceState(Activity.java:1102)
        at android.app.Activity.performRestoreInstanceState(Activity.java:1057)
        at android.app.Instrumentation.callActivityOnRestoreInstanceState(Instrumentation.java:1260)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2751)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2856)
        at android.app.ActivityThread.-wrap11(Unknown Source:0)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1589)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loop(Looper.java:164)
        at android.app.ActivityThread.main(ActivityThread.java:6494)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:438)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807)
     Caused by: java.lang.ClassNotFoundException: com.avito.android.krop.ZoomableImageView$SavedState
        at java.lang.Class.classForName(Native Method)
        at java.lang.BootClassLoader.findClass(ClassLoader.java:1355)
        at java.lang.BootClassLoader.loadClass(ClassLoader.java:1415)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
        at java.lang.Class.classForName(Native Method) 
        at java.lang.Class.forName(Class.java:453) 
        at android.os.Parcel.readParcelableCreator(Parcel.java:2897) 
        at android.os.Parcel.readParcelable(Parcel.java:2851) 
        at com.avito.android.krop.KropView$SavedState.<init>(KropView.kt:263) 
        at com.avito.android.krop.KropView$SavedState$CREATOR.createFromParcel(KropView.kt:268) 
        at com.avito.android.krop.KropView$SavedState$CREATOR.createFromParcel(KropView.kt:266) 
        at android.os.Parcel.readParcelable(Parcel.java:2860) 
        at android.os.Parcel.readValue(Parcel.java:2754) 
        at android.os.Parcel.readSparseArrayInternal(Parcel.java:3204) 
        at android.os.Parcel.readSparseArray(Parcel.java:2416) 
        at android.os.Parcel.readValue(Parcel.java:2811) 
        at android.os.Parcel.readArrayMapInternal(Parcel.java:3123) 
        at android.os.BaseBundle.initializeFromParcelLocked(BaseBundle.java:273) 
        at android.os.BaseBundle.unparcel(BaseBundle.java:226) 
        at android.os.Bundle.getSparseParcelableArray(Bundle.java:1009) 
        at com.android.internal.policy.PhoneWindow.restoreHierarchyState(PhoneWindow.java:2129) 
        at android.app.Activity.onRestoreInstanceState(Activity.java:1102) 
        at android.app.Activity.performRestoreInstanceState(Activity.java:1057) 
        at android.app.Instrumentation.callActivityOnRestoreInstanceState(Instrumentation.java:1260) 
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2751) 
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2856) 
        at android.app.ActivityThread.-wrap11(Unknown Source:0) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1589) 
        at android.os.Handler.dispatchMessage(Handler.java:106) 
        at android.os.Looper.loop(Looper.java:164) 
        at android.app.ActivityThread.main(ActivityThread.java:6494) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:438) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807) 
     Caused by: java.lang.NoClassDefFoundError: Class not found using the boot class loader; no stack trace available
```

--- a/krop/src/main/java/com/avito/android/krop/KropView.kt
+++ b/krop/src/main/java/com/avito/android/krop/KropView.kt
@@ -260,7 +260,7 @@ class KropView(context: Context, attrs: AttributeSet) : FrameLayout(context, att
             aspectY = source.readInt()
             overlayColor = source.readInt()
             overlayShape = source.readInt()
-            imageViewState = source.readParcelable(Parcelable::class.java.classLoader)
+            imageViewState = source.readParcelable(SavedState::class.java.classLoader)
         }
 
         companion object CREATOR : Parcelable.Creator<SavedState> {

